### PR TITLE
Update/version 4.3.3

### PR DESCRIPTION
**Pull request by AcidSepp**
Changed 'dc-square' to 'HiveMQ'
added HelloWorldInterceptor and a corresponding integration test using the HiveMQ Testcontainer and the HiveMQ MQTT client.

Do not merge before the release of the HiveMQ Testcontainer 1.1.0.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
     <groupId>com.hivemq</groupId>
     <artifactId>hivemq-extension-archetype</artifactId>
-    <version>4.3.0</version>
+    <version>4.3.3</version>
```
Comment (SgtSilvio) [inline pom.xml:8]: the version should be 4.3.1 as this has nothing to do with the current HiveMQ patch version

File: HEADER [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
```
Comment (SgtSilvio) [inline HEADER:1]: the header should not contain any language specific comment characters

File: README.adoc [hunk 3]
```diff
@@ -17,7 +17,7 @@ A new HiveMQ extension project can be created by running the following maven com
 
 [source,bash]
 ----
-mvn archetype:generate -DarchetypeGroupId=com.hivemq -DarchetypeArtifactId=hivemq-extension-archetype -DarchetypeVersion=4.3.0
+mvn archetype:generate -DarchetypeGroupId=com.hivemq -DarchetypeArtifactId=hivemq-extension-archetype -DarchetypeVersion=4.3.3
```
Comment (SgtSilvio) [inline README.adoc:20]:
```suggestion
mvn archetype:generate -DarchetypeGroupId=com.hivemq -DarchetypeArtifactId=hivemq-extension-archetype -DarchetypeVersion=4.3.1
```

File: pom.xml (hunk 1 above)
Comment (SgtSilvio) [inline pom.xml:8]:
```suggestion
    <version>4.3.1</version>
```

File: pom.xml [hunk 4]
```diff
@@ -19,16 +19,22 @@
...
 
     <developers>
         <developer>
             <name>Florian Limpöck</name>
-            <email>florian.limpoeck@dc-square.de</email>
-            <organization>dc-square GmbH</organization>
-            <organizationUrl>http://www.dc-square.de</organizationUrl>
+            <email>florian.limpoeck@hivemq.com</email>
+            <organization>HiveMQ GmbH</organization>
+            <organizationUrl>http://www.hivemq.com</organizationUrl>
```
Comment (SgtSilvio) [inline pom.xml:31]:
```suggestion
            <organizationUrl>https://www.hivemq.com</organizationUrl>
```

File: pom.xml [hunk 5]
```diff
@@ -19,16 +19,22 @@
...
+        </developer>
+        <developer>
+            <name>Yannick Weber</name>
+            <email>yannick.weber@hivemq.com</email>
+            <organization>HiveMQ GmbH</organization>
+            <organizationUrl>http://www.hivemq.com</organizationUrl>
```
Comment (SgtSilvio) [inline pom.xml:37]:
```suggestion
            <organizationUrl>https://www.hivemq.com</organizationUrl>
```

File: src/main/resources/archetype-resources/README.adoc [hunk 6]
```diff
@@ -4,19 +4,22 @@
...
 :hivemq-support: http://www.hivemq.com/support/
+:hivemq-testcontainer: https://github.com/hivemq/hivemq-testcontainer
+:hivemq-mqtt-client: https://github.com/hivemq/hivemq-mqtt-client
 
 == HiveMQ 4 Hello World Extension
 
 *Type*: Demonstration Extension
 
-*Version*: 4.3.0
+*Version*: 4.3.3
```
Comment (SgtSilvio) [inline src/main/resources/archetype-resources/README.adoc:14]:
```suggestion
*Version*: 4.3.1
```

**Review by RobinAtherton — APPROVED** [review]

**Pull request closed without merging**